.storybook/preview.js

```diff
@@ -181,13 +181,15 @@ const StoryWrap = styled.div`
   background-color: ${({ theme }) => theme.color.surface};
 `;
 
-addDecorator(renderStory => (
+addDecorator(Story => (
```

Comment:
This change allowed me to use hooks in our stories

Reply:
Ahh, I see. Very nice. 👏🏻